pyproject.toml

```diff
@@ -44,3 +44,8 @@ docstring-convention = "google"
 copyright-check = "True"
 copyright-author = "Canonical Ltd."
 copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
+
+[tool.mypy]
```

Comment:
We have some additional settings defined in this standard: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints, should they be added too?

Reply:
There's an item in the backlog to address linting properly. IMHO this is out of scope